README.md

```diff
@@ -1,75 +1,137 @@
+polls applications.
+
+## Project Creation
+
+The basic skeleton of this project was created using the standard Django commands:
+
+   ```
+   django-admin startproject mysite
+   ```
+
+The DATABASE field in the myproject/settings.py is modified to use CloudSQL locally, and through
+a Unix socket when deployed to AppEngine. The static file handler is also set up 
+in the app.yaml.
+
+## Enable Cloud SQL
+
+This project assumes you are using [CloudSQL](https://cloud.google.com/sql/docs/introduction)
+as your database. CloudSQL is a managed MySQL instance hosted by Google Cloud, although you
+could also host a MySQL instance elsewhere and use that. For better scalability, consider
+[Cloud Datastore](https://cloud.google.com/datastore/docs/concepts/overview?hl=en), although
+you must use a custom backend such as [Djangae](https://github.com/potatolondon/djangae) for your
+models to work with it.
 
 ## Run Locally
 1. Install the [App Engine Python SDK](https://developers.google.com/appengine/downloads).
-See the README file for directions. You'll need Python 2.7, [pip 1.4 or later](http://www.pip-installer.org/en/latest/installing.html), and [mercurial](http://mercurial.selenic.com/) (`hg`) installed too.
-If you're missing `hg` you can install it with `pip install mercurial` once you have `pip` installed.
+See the README file for directions. You'll need python 2.7 and [pip 1.4 or later](http://www.pip-installer.org/en/latest/installing.html) installed too.
 
-2. Clone this repo with:
+1. Clone this repo with
 
    ```
    git clone https://github.com/GoogleCloudPlatform/appengine-django-skeleton.git
    ```
-3. Install dependencies in the project's directory - App Engine
-   can only import libraries from inside your project directory.
-   ```
-   cd appengine-django-skeleton
+1. Install dependencies in the project's lib directory.
+   Note: App Engine can only import libraries from inside your project directory. The pip -t flag installs dependencies into the
+   directory specified, so here it's used to install the lastest Django version into lib/.
 
-   ./build.sh
    ```
-4. Run this project locally from the command line:
+   cd appengine-try-python-django
+   pip install -r requirements.txt -t lib
+   ```
+1. Create a new CloudSQL instance. 
+    * From the [Google Cloud Console](http://console.developer.google.com), go to Storage > CloudSQL> Create Instance
+    * Under Acces Control > IP Address > Request IPv4 Address. This address will be your HOST for remote access to the 
+      CloudSQL instance in mysite/settings.py, so replace `<your-database-host>` with this address.
+    * Under Databases, click New database and create the name for your database in mysite/settings.py. Replace
+      `<your-database-name>` with this value.
+    * Under Access Control > Authorization > Allowed Networks, click Add item, and add Network 0.0.0.0/0. This opens up
```

Comment:
See if you can deep link this. Also mention this is only necessary if you want to connect to the cloudsql instance locally.